transaction-relayer/src/main.rs

```diff
@@ -511,9 +549,8 @@ fn main() {
         delay_packet_receiver,
         leader_cache.handle(),
         public_ip,
-        (args.tpu_quic_port..args.tpu_quic_port + args.num_tpu_quic_servers as u16).collect(),
-        (args.tpu_quic_fwd_port..args.tpu_quic_fwd_port + args.num_tpu_fwd_quic_servers as u16)
-            .collect(),
+        (args.tpu_quic_port..args.tpu_quic_port + args.num_tpu_quic_servers).collect(),
```

Comment:
imo should grab the ports above + use here instead of rebuilding it. could result in mismatch state if logic upstream changes